pom.xml

```diff
@@ -41,30 +39,30 @@
   </developers>
 
   <properties>
-    <apache-maven.version>3.3.9</apache-maven.version>
-    <mavenPluginPlugin.version>3.4</mavenPluginPlugin.version>
-    <slf4j.version>1.7.5</slf4j.version>
+    <apache-maven.version>3.6.3</apache-maven.version>
+    <mavenPluginPlugin.version>3.6.0</mavenPluginPlugin.version>
+    <slf4j.version>1.7.30</slf4j.version>
     <junit.version>4.11</junit.version>
-    <guava.version>19.0</guava.version>
-    <eclipse-sisu.version>0.3.2</eclipse-sisu.version>
-    <plugin-testing.version>2.9.2</plugin-testing.version>
+    <guava.version>28.2-jre</guava.version>
+    <eclipse-sisu.version>0.3.4</eclipse-sisu.version>
+    <plugin-testing.version>2.9.3-SNAPSHOT</plugin-testing.version>
```

Comment:
@mosabua FYI: This needs an update once https://github.com/takari/takari-plugin-testing-project/pull/22 is merged and released.